docs/content/tooling/coverage.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Coverage
+description: How to generate a code coverage report
+status: Stable
+source: https://github.com/xedi/sdk-js/tree/master/testing
```

Comment:
Is this necessary? I'm torn. I see the value but at the same time it feels out of place... /shrug